DistributionUpdate/PackageUpdate/currentPackageInfoURLList

```diff
@@ -160,7 +160,7 @@ Thelma      https://gap-packages.github.io/Thelma/PackageInfo.g
 TomLib      https://gap-packages.github.io/tomlib/PackageInfo.g
 ToolsForHomalg https://homalg-project.github.io/homalg_project/ToolsForHomalg/PackageInfo.g
 toric       https://gap-packages.github.io/toric/PackageInfo.g
-ToricVarieties https://homalg-project.github.io/homalg_project/ToricVarieties/PackageInfo.g
+ToricVarieties MOVE https://homalg-project.github.io/ToricVarieties_project/ToricVarieties/PackageInfo.g
```

Comment:
Thanks @HereAround. Just to check, is it still correct that URLs in
```
SourceRepository := rec( 
  Type := "git", 
  URL := "https://github.com/homalg-project/homalg_project"
),
IssueTrackerURL := Concatenation( ~.SourceRepository.URL, "/issues" ),
PackageWWWHome := "https://homalg-project.github.io/homalg_project/ToricVarieties/",
ArchiveFormats := ".tar.gz .zip",
ArchiveURL     := Concatenation( "https://github.com/homalg-project/homalg_project/releases/download/ToricVarieties-", ~.Version, "/ToricVarieties-", ~.Version ),
```
point to `homalg-project/homalg_project` repository?

Reply:
Thank you for pointing this out. I have updated the PackageInfo.g with the new URL.